po/ar.po

```diff
@@ -347,129 +345,129 @@ msgstr "تمرين: فيبوناتشي"
 #: src/SUMMARY.md src/android/aidl/types/primitives.md
 #: src/android/interoperability/cpp/type-mapping.md
 #, fuzzy
 msgid "`String`"
-msgstr "السلاسل النص او الكتابة (Strings)"
+msgstr "السلاسل النصية (Strings)"
 
 #: src/SUMMARY.md src/std-types/vec.md
 msgid "`Vec`"
-msgstr ""
+msgstr "المصفوفات (Vectors)"
 
 #: src/SUMMARY.md src/std-types/hashmap.md src/bare-metal/no_std.md
 msgid "`HashMap`"
-msgstr ""
+msgstr "خريطة المشفرة (HashMap)"
 
 #: src/SUMMARY.md src/std-types/exercise.md
 #, fuzzy
 msgid "Exercise: Counter"
-msgstr "تمرين: فيبوناتشي"
+msgstr "تمرين: عداد"
 
 #: src/SUMMARY.md src/std-traits.md
 msgid "Standard Library Traits"
-msgstr ""
+msgstr "سمات المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-traits/comparisons.md
 #: src/concurrency/welcome-async.md
 msgid "Comparisons"
-msgstr ""
+msgstr "مقارنات"
 
 #: src/SUMMARY.md src/std-traits/operators.md
 msgid "Operators"
-msgstr ""
+msgstr "المشغلات (`Operators`)"
 
 #: src/SUMMARY.md src/std-traits/from-and-into.md
 msgid "`From` and `Into`"
-msgstr ""
+msgstr "السمات (من و إلى) (From and Into)"
 
 #: src/SUMMARY.md src/std-traits/casting.md
 msgid "Casting"
-msgstr ""
+msgstr "التحويل"
 
 #: src/SUMMARY.md src/std-traits/read-and-write.md
 msgid "`Read` and `Write`"
-msgstr ""
+msgstr "السمات (القراءة و الكتابة) (Read and Write)"
 
 #: src/SUMMARY.md
 msgid "`Default`, struct update syntax"
-msgstr ""
+msgstr "الإفتراضي، بناء جملة تحديث البنية (Default, struct update syntax)"
 
 #: src/SUMMARY.md src/std-traits/closures.md
 msgid "Closures"
-msgstr ""
+msgstr "الإغلاقات"
 
 #: src/SUMMARY.md src/std-traits/exercise.md
 msgid "Exercise: ROT13"
-msgstr ""
+msgstr "تمرين: ROT13"
 
 #: src/SUMMARY.md
 #, fuzzy
 msgid "Day 3: Morning"
-msgstr "اليوم الأول: صباحا"
+msgstr "اليوم الثالث: صباحا"
 
 #: src/SUMMARY.md src/memory-management.md
 msgid "Memory Management"
-msgstr ""
+msgstr "إدارة الذاكرة"
 
 #: src/SUMMARY.md src/memory-management/review.md
 msgid "Review of Program Memory"
-msgstr ""
+msgstr "مراجعة ذاكرة البرنامج"
 
 #: src/SUMMARY.md src/memory-management/approaches.md
 msgid "Approaches to Memory Management"
-msgstr ""
+msgstr "طرق إدارة الذاكرة"
 
 #: src/SUMMARY.md src/memory-management/ownership.md
 msgid "Ownership"
-msgstr ""
+msgstr "الملكية"
 
 #: src/SUMMARY.md src/memory-management/move.md
 msgid "Move Semantics"
-msgstr ""
+msgstr "دلالات النقل"
 
 #: src/SUMMARY.md
 msgid "`Clone`"
-msgstr ""
+msgstr "الاستنساخ"
 
 #: src/SUMMARY.md src/memory-management/copy-types.md
 msgid "Copy Types"
-msgstr ""
+msgstr "أنواع النسخ"
 
 #: src/SUMMARY.md
 msgid "`Drop`"
-msgstr ""
+msgstr "الإسقاط"
 
 #: src/SUMMARY.md src/memory-management/exercise.md
 msgid "Exercise: Builder Type"
-msgstr ""
+msgstr "تمرين: نوع الباني"
 
 #: src/SUMMARY.md src/smart-pointers.md
 msgid "Smart Pointers"
-msgstr ""
+msgstr "المؤشرات الذكية"
 
 #: src/SUMMARY.md src/smart-pointers/box.md
 #: src/android/interoperability/cpp/type-mapping.md
 msgid "`Box<T>`"
-msgstr ""
+msgstr "صندوق<T>"
```

Comment:
I think صندوق is more of a literal translation. Given the meaning go Box and documentation https://doc.rust-lang.org/std/boxed/struct.Box.html I more inclined to use مؤشر مالك

Reply:
Since this is in backticks, it is code. So it should not be translated. It is the heading shown here: [20.1. Box<T>](https://google.github.io/comprehensive-rust/smart-pointers/box.html).

To "not translate" a string, you just copy the string from the `msgid` field to the `msgstr` field.

Reply:
```suggestion
msgstr "`Box<T>`"
```